io.openems.edge.goodwe.et/src/io/openems/edge/goodwe/et/FeneconGoodweET.java

```diff
@@ -0,0 +1,1104 @@
+package io.openems.edge.goodwe.et;
+
+import org.osgi.service.cm.ConfigurationAdmin;
+import org.osgi.service.component.ComponentContext;
+import org.osgi.service.component.annotations.Activate;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.ConfigurationPolicy;
+import org.osgi.service.component.annotations.Deactivate;
+import org.osgi.service.component.annotations.Reference;
+import org.osgi.service.component.annotations.ReferenceCardinality;
+import org.osgi.service.component.annotations.ReferencePolicy;
+import org.osgi.service.component.annotations.ReferencePolicyOption;
+import org.osgi.service.metatype.annotations.Designate;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import io.openems.common.types.OpenemsType;
+import io.openems.edge.bridge.modbus.api.AbstractOpenemsModbusComponent;
+import io.openems.edge.bridge.modbus.api.BridgeModbus;
+import io.openems.edge.bridge.modbus.api.ElementToChannelConverter;
+import io.openems.edge.bridge.modbus.api.ModbusProtocol;
+import io.openems.edge.bridge.modbus.api.element.BitsWordElement;
+import io.openems.edge.bridge.modbus.api.element.DummyRegisterElement;
+import io.openems.edge.bridge.modbus.api.element.FloatDoublewordElement;
+import io.openems.edge.bridge.modbus.api.element.SignedWordElement;
+import io.openems.edge.bridge.modbus.api.element.StringWordElement;
+import io.openems.edge.bridge.modbus.api.element.UnsignedDoublewordElement;
+import io.openems.edge.bridge.modbus.api.element.UnsignedWordElement;
+import io.openems.edge.bridge.modbus.api.task.FC16WriteRegistersTask;
+import io.openems.edge.bridge.modbus.api.task.FC3ReadRegistersTask;
+import io.openems.edge.common.component.OpenemsComponent;
+import io.openems.edge.common.sum.GridMode;
+import io.openems.edge.common.taskmanager.Priority;
+import io.openems.edge.common.type.TypeUtils;
+import io.openems.edge.ess.api.AsymmetricEss;
+import io.openems.edge.ess.api.SymmetricEss;
+import io.openems.edge.ess.dccharger.api.EssDcCharger;
+import io.openems.edge.ess.power.api.Power;
+
+@Designate(ocd = Config.class, factory = true)
+@Component(//
+		name = "Goodwe.ET.Battery.Inverter", //
+		immediate = true, //
+		configurationPolicy = ConfigurationPolicy.REQUIRE) //
+
+public class FeneconGoodweET extends AbstractOpenemsModbusComponent
+		implements AsymmetricEss, SymmetricEss, EssDcCharger, OpenemsComponent {
+
+	@SuppressWarnings("unused")
+	private final Logger log = LoggerFactory.getLogger(FeneconGoodweET.class);
+	@SuppressWarnings("unused")
+	private boolean readonly = false;
+
+	public static final int DEFAULT_UNIT_ID = 5;
```

Comment:
Here you are using default id 5; above you used 1. None of them is the default unit id value according to GoodWe docs